src/base/display.jl

```diff
@@ -31,43 +38,58 @@ function Base.show(io::IO, dataset::AbstractDataset)
         println(io, "\nNumber of feature layers: $nlayers")
         ndisplay = min(nlayers, 5) # display up to 5 layers
```

Comment:
That 5 could be a constant outside of this function? Note that we can also check for compactness with `compact = get(io, :compact, false)` and we could print all elements when given a plaintext mimetype, like `::MIME"text/plain"` as second argument to show. See this [thread](https://discourse.julialang.org/t/show-and-showcompact-on-custom-types/8493/10).

Reply:
Opened #203 for it